Src/cql-lm/schema/elm/cqlannotations.xsd

```diff
@@ -1,24 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns="urn:hl7-org:cql-annotations:r1" xmlns:sch="http://purl.oclc.org/dsdl/schematron" targetNamespace="urn:hl7-org:cql-annotations:r1" elementFormDefault="qualified" attributeFormDefault="unqualified">
+	<xs:complexType name="CqlToElmBase" abstract="true">
```

Comment:
Need to make sure these changes are also synced to the cql ig repo:

https://cql.hl7.org/elm.html

https://github.com/HL7/cql

Reply:
Should I do a PR there? I am not sure what are the procedures to change that repo.

Reply:
hum.. I realized that the [expression.xsd at HL7/CQL](https://github.com/HL7/cql/blob/master/spec/elm/schema/expression.xsd) is not in sync with the one here.

Besides many documentation changes, these two types are not in the HL7/CQL repo

```xml	
<xs:complexType name="ParameterTypeSpecifier">
<xs:annotation>
	<xs:documentation>A type which is generic class parameter such as T in MyGeneric&lt;T extends SomeType&gt;.</xs:documentation>
</xs:annotation>
<xs:complexContent>
	<xs:extension base="TypeSpecifier">
		<xs:attribute name="parameterName" type="xs:string" use="required"/>
	</xs:extension>
</xs:complexContent>
</xs:complexType>
```

Reply:
@brynrhodes - Do we need to push those new types upstream to the CQL repo?

Reply:
FYI, PR has been created: https://github.com/HL7/cql/pull/76